problems/strings.js

```diff
@@ -87,14 +114,33 @@ function removeLastWord() {}
  * @returns {string}
  */
 
-function everyOtherLetter() {}
-
+function everyOtherLetter(string) {
+  let even = [];
+  let odd = [];
+  for(let i = 0; i < string.length; i++){
+    if( (i%2) != 0 ){
+      odd += string[i]
+} else{
+even += string[i]
+    }
+}
+return even
+}
 /**
  * Takes in a string and returns a new string that has all the vowels removed.
  * @param {string}
  * @returns {string}
  */
-function disemvowel() {}
+function disemvowel(string) {
+  let newString = "";
+  for (let i = 0; i < string.length; i++) {
+    let char = string[i];
+    if (char !== "a" && char !== "e" && char !== "i" && char !== "o" && char !== "u" && char !=="A" && char !=="E" && char !=="I" && char !=="O" && char !=="U") {
```

Comment:
I want you to look up the .includes() method to make for a much cleaner version of this